torch/csrc/jit/script/module.h

```diff
@@ -239,13 +233,11 @@ struct TORCH_API Module {
   c10::ArrayRef<Slot> get_attributes() const {
     return attributes_;
   }
-  const std::vector<std::unique_ptr<Method>>& get_methods() const {
-    // force methods_ to be up to date by querying all
-    // methods.
-    for (const auto& m : class_compilation_unit().get_functions()) {
-      get_method(m->name());
-    }
-    return methods_;
+  const std::vector<Method> get_methods() const {
+    return fmap(class_compilation_unit().get_functions(),
+                [&](const std::shared_ptr<Function> &func) {
+                  return Method(const_cast<Module *>(this), func);
```

Comment:
Can we remove this const_cast? It's not clear to me why methods need a mutable reference to their owner anyway